packages/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.nuspec

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.SqlTools.ManagedBatchParser</id>
+        <version>1.0.0</version>
+        <authors>Microsoft</authors>
+        <projectUrl>https://github.com/Microsoft/sqltoolsservice/</projectUrl>
+        <license type="expression">MIT</license>
+        <iconUrl>http://s.gravatar.com/avatar/6f39d8562df0a3509a8240fb281bc5bd?s=80</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Managed Batch parser for Sql Tools Services.</description>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <tags>SQL Tools XPLAT Managed Batch Parser</tags>
+        <dependencies>
+            <group targetFramework="net472">
+                <dependency id="Microsoft.Data.SqlClient" version="5.0.1" />
```

Comment:
Another reason to use csproj, not having to remember to update this if these dependencies ever change...

If you're going to keep this for now though please put these packages in a separate section in the Packages.props and add a comment telling people to update this nuspec as well if those ever change